.github/workflows/renovate.yml

```
@@ -0,0 +1,31 @@
+name: Renovate
+on:
+  # Allows manual/automated trigger for debugging purposes
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: "Override default log level"
+        required: false
+        default: "info"
+        type: string
+      overrideSchedule:
+        description: "Override all schedules"
+        required: false
+        default: "false"
+        type: string
+  schedule:
+    - cron: '0 8 * * *'
+jobs:
+  renovate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.1.1
+      # Don't waste time starting Renovate if JSON is invalid
+      - name: Validate Renovate JSON
+        run:  npx --yes --package renovate -- renovate-config-validator
+      - name: Self-hosted Renovate
+        uses: renovatebot/github-action@v40.1.5
+        with:
+          configurationFile: .github/renovate.json5
+          token: ${{ secrets.RENOVATE_TOKEN }}
```

This should work, we should be able to create PRs with that:
```suggestion
          token: ${{ secrets.GITHUB_TOKEN }}
```

---

no, that won't have all the required access, I'll see if we manage to create an app on monday

---

Hi @phisco any updates on this?

---

Sorry, got lost, I'll make sure to check this out next week 😅